frontend/packages/gitops-plugin/src/components/details/GitOpsResourcesSection.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Stack,
+  StackItem,
+  Split,
+  SplitItem,
+  Card,
+  CardBody,
+  Tooltip,
+} from '@patternfly/react-core';
+import { ResourceIcon } from '@console/internal/components/utils';
+import { HeartBrokenIcon } from '@patternfly/react-icons';
+import { GitOpsEnvironmentService } from '../utils/gitops-types';
+import './GitOpsResourcesSection.scss';
+
+interface GitOpsResourcesSectionProps {
+  services: GitOpsEnvironmentService[];
+}
+
+const GitOpsResourcesSection: React.FC<GitOpsResourcesSectionProps> = ({ services }) => {
```

Comment:
There are a lot of static values in this component, i take it that this is temporary content?

Reply:
yes, that content is just temporary! It will be changed once we have backend changes in place